src/map/mob.cpp

```diff
@@ -4361,6 +4360,28 @@ bool MobDatabase::parseDropNode(std::string nodeName, const ryml::NodeRef& node,
 	return true;
 }
 
+/**
+ * Mob DB constructor
+ */
+s_mob_db::s_mob_db()
+{
+	status.max_hp = 1;
+	status.max_sp = 1;
+	status.str = 1;
+	status.agi = 1;
+	status.vit = 1;
+	status.int_ = 1;
+	status.dex = 1;
+	status.luk = 1;
+	status.ele_lv = 1;
+	status.speed = DEFAULT_WALK_SPEED;
+	status.adelay = cap_value(0, battle_config.monster_max_aspd * 2, 4000);
+	status.amotion = cap_value(0, battle_config.monster_max_aspd, 2000);
+	status.mode = static_cast<e_mode>(MONSTER_TYPE_06);
```

Comment:
if only we had [aggregate initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization), need c++20 though.